### DISCUSSION TITLE
Homework Part 1

### DISCUSSION THREAD

**Discussion by mattlewis10** [cooperrc/computational-mechanics#226] [General]
How do I download the transistor data csv file for Part 1?

**Comment by cooperrc** [comment]
If you're working in the compmech.uconn.edu server its already in the data folder:

`pd.read_csv('/home/jovyan/work/computational-mechanics/data)`

If you want to download the csv for other reasons its here:
[transistor_data.csv](https://github.com/cooperrc/computational-mechanics/blob/master/data/transistor_data.csv)

There's an option to download the file in the top right corner
